code/game/machinery/syndicatebeacon.dm

```diff
@@ -49,12 +49,11 @@
 			src.updateUsrDialog()
 			return
 		charges -= 1
-		switch(rand(1,2))
-			if(1)
-				temptext = "<font color=red><i><b>Double-crosser. You planned to betray us from the start. Allow us to repay the favor in kind.</b></i></font>"
-				src.updateUsrDialog()
-				spawn(rand(50,200)) selfdestruct()
-				return
+		if(prob(50))
+			temptext = "<font color=red><i><b>Double-crosser. You planned to betray us from the start. Allow us to repay the favor in kind.</b></i></font>"
+			updateUsrDialog()
+			spawn(rand(50,200)) selfdestruct()
```

Comment:
Any chance you can make this a timer at the same time? 

`spawn()` needs to be shot.

Reply:
Yeah I can do that.